pkg/cmd/grafana-cli/utils/grafana_path.go

```diff
@@ -23,7 +23,7 @@ func GetGrafanaPluginDir(currentOS string) string {
 func getGrafanaRoot() (string, error) {
 	ex, err := os.Executable()
 	if err != nil {
-		return "", xerrors.New("Failed to get executable path")
+		return "", errutil.Wrap("failed to get executable path", err)
```

Comment:
It is already checked that the `err != nil` so I think there is no reason to use `errutil.Wrap`; `fmt.Errorf` should be enough.

Reply:
@papagian I think there must be a misunderstanding. `errutil.Wrap` doesn't do any checking, it wraps `err` with another message ("failed to get executable path"). Now, we can wrap also with `fmt.Errorf`, but so far we have been using `errutil.Wrap` in Grafana code. Shall we stop using `errutil,.Wrap`?

Reply:
It's not this [one](https://github.com/grafana/grafana/blob/f8d4d3fd07bd78003445908c484a012d82bf6b58/pkg/util/errutil/errors.go#L8)?

Reply:
@papagian Yes, but I don't see the relevance? Why exactly do you think wrapping isn't necessary?

Reply:
Ah, you're referring to that `errutil.Wrap` checks if err is `nil` at the beginning, but that isn't its real functionality.

Reply:
yes, my point is that this could be replaced with:
`fmt.Errorf("failed to get executable path: %w", err)`

So actually it's the same discussion whether there is need to keep `errutil.Wrap` at all.